src/core/controller.h

```diff
@@ -35,12 +35,9 @@ class Controller : public QObject
 
     Controller(const Controller&) = delete;
     ~Controller();
-    void operator=(const Controller&) = delete;
```

Comment:
Why is this line removed? It seems like the copy assignment for this class might need to be explicitly marked delete or we should implement the copy/ assignment operators. I don't think the default ones will be correct. (Nor do I think this class should be assigned  or copied).

Reply:
https://doc.qt.io/qt-5/qobject.html#no-copy-constructor-or-assignment-operator

Reply:
Oh, thanks.. Today I learned.